innersource-license.md

```diff
@@ -0,0 +1,94 @@
+
+## Problem
+
+When two or more legal entities within an organization want to share code with each other, they need an agreement about the terms and often a legal contract. Creating such agreements on a per project basis takes effort and creates a barrier for sharing. i.e. a team within a legal entity might decide not to share their source code with another legal entity in the organization because it seems complicated.
+
+Barriers for sharing can lead to silos and duplication of effort in rebuilding similar solutions in multiple parts of the organization.
+
+At the time of sharing the source code, it can not be reliably predicted what the value of sharing will be. If the activity of sharing requires significant effort (i.e. negotiating terms for the usage), the legal entities are less likely to do it, as they are concerned about the return on investment.
+
+## Context
+
+- A large organization with many legal entities (subsidiaries) that want to share code. When the organization gets larger, the value of this pattern increases.
+- As per definition, the legal entities have their own legal rights and obligations.
+- Multiple of these legal entities are developing software, and are using services of the other legal entities. They have a motivation to contribute to each other’s source code.
+- A sufficient complexity of the organization and its organizational structure
+
+## Forces
+
+- **Level of effort** required to write formal agreements, especially if they need to take into account technical, legal, and business perspectives.
+- A large organization (consisting of many legal entities) has many **internal regulations**. Any new agreements that are made have to comply with these regulations, e.g. security, privacy, procurement processes, etc. The volume of regulations can make it difficult to assess whether sharing software between two legal entities is compliant with these regulations, especially when there is no standard procedure.
+- If any of the legal entities in the organization has a **business model** that depends on proprietary code and accounting of licensing fees within the organization
+- **Company culture** that isn’t used to InnerSource collaboration and sharing code. This results in uncertainty about the rights and obligations when using shared code.
+- Freedom over using the software leads to competition, and spread of ownership
+- There are legal contracts in place which cover the sharing of source code. These contracts are not standardized, so they create additional effort in negotiating and understanding for every project. The existing contracts may also not allow sharing source code in an open enough sense to support a true InnerSource approach.
+- Alternatively, there are no legal contracts in place but source code is shared informally. That might create uncertainty in cases where clarity about ownership and rights and obligations is needed.
```

Comment:
This one is at least partially a duplication of the force above, regarding the consequences of _Freedom over using the software"_